<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>OpsSatellites</class>
 <widget class="QWidget" name="OpsSatellites">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>468</width>
    <height>504</height>
   </rect>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="1" column="0">
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QCheckBox" name="kcfg_ShowSatellites">
         <property name="text">
          <string>Show satellites</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="kcfg_ShowVisibleSatellites">
         <property name="text">
          <string>Show only visible satellites</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QCheckBox" name="kcfg_ShowSatellitesLabels">
         <property name="text">
          <string>Show labels</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="kcfg_DrawSatellitesLikeStars">
         <property name="text">
          <string>Draw satellites like stars</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="KPushButton" name="UpdateTLEButton">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>Update TLEs</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QTreeWidget" name="SatListTreeView">
       <property name="rootIsDecorated">
        <bool>true</bool>
       </property>
       <property name="headerHidden">
        <bool>true</bool>
       </property>
       <property name="columnCount">
        <number>1</number>
       </property>
       <attribute name="headerVisible">
        <bool>false</bool>
       </attribute>
       <attribute name="headerCascadingSectionResizes">
        <bool>false</bool>
       </attribute>
       <attribute name="headerDefaultSectionSize">
        <number>60</number>
       </attribute>
       <column>
        <property name="text">
         <string notr="true">1</string>
        </property>
       </column>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>KPushButton</class>
   <extends>QPushButton</extends>
   <header>kpushbutton.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
